src/api/spec/controllers/webui/package_controller_spec.rb

```diff
@@ -264,7 +264,8 @@ def remove_file_post
   end
 
   describe 'GET #revisions' do
-    let(:package) { create(:package_with_revisions, name: 'package_with_one_revision', revision_count: 1, project: source_project) }
```

Comment:
The specs were previously relying on a package in `source_project`, which the home project of `user`. Preventing a user from accessing the source of their own home project isn't possible anymore.